internal/datanode/allocator.go

```diff
@@ -28,13 +28,15 @@ type allocatorInterface interface {
 }
 
 type allocator struct {
+	sourceID      int64
```

Comment:
`allocator` should not rely on `sourceID`, it makes no sense to allocator.

Reply:
But in fact, these grpc requests do depend on the server ID, and the global Params variable only hide this dependency, but do not eliminate this dependency. For convenience to using global variables, I think it is unreliable. And the initialization sequence of global variables requires special attention to prevent secondary assignments. At the same time, the serverID and Params.NodeID of the session are variables with the same value and different names. If the two exist at the same time, the developer will have ambiguity in use.